tests/vm_test.go

```diff
@@ -769,10 +591,12 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		DescribeTable("should report an error status when VM scheduling error occurs", func(unschedulableFunc func(vmi *v1.VirtualMachineInstance)) {
-			vmi := tests.NewRandomVMIWithEphemeralDisk("no-such-image")
+			vmi := libvmi.New(
+				libvmi.WithContainerImage("no-such-image"),
+				libvmi.WithResourceMemory("128Mi"),
+			)
 			unschedulableFunc(vmi)
-
-			vm := createVirtualMachine(true, vmi)
+			vm := createVM(vmi, true)
```

Comment:
boolean arguments are problematic to the reader. what's so "true" here, they may ask. A call to `createRunningVM` would be clearer.